src/lib/y2users/config.rb

```diff
@@ -102,6 +102,7 @@ def copy
 
       config = self.class.new
       config.attach(elements)
+      config.useradd_config = useradd_config.dup
```

Comment:
What happens when merging configs? How is this object merged?

Reply:
I haven't though about it. I wonder whether it's actually relevant. Can you describe the "use case"?

Reply:
Maybe when using AutoYaST in a running system. The system config will be merged with the config from the AutoYaST profile, and both configs will have their own useradd conf.

Reply:
May be. But that's something to be solved at https://trello.com/c/QU2gDrTK/2486-3-autoyast-userdefaults-section (already planned for this sprint)